elements/rh-sass/variables/_colors.scss

```diff
@@ -99,6 +107,14 @@ $rh-color--surface--darker--link--visited:     $rh-color--ui-link--on-dark--visi
 $rh-color--surface--darker--link--hover:       $rh-color--ui-link--on-dark--hover !default;
 $rh-color--surface--darker--link--focus:       $rh-color--ui-link--on-dark--focus !default;
 
+// Dark group - @TODO this is a dupe of lighter until lighter can be deprecated
```

Comment:
I was basing it off of the color properties that card uses which are "darkest" and "dark" but since our properties are "darkest" and "darker" - I think we should be consistent one way or the other.